packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -385,7 +385,8 @@ Occasionally you may want to verify that the device secret is still valid by usi
 {
     "active": true,
     "sid": "102oRgEWx5lRTWHilEoGfed4w",
-    "token_type": "urn:x-oath:params:oauth:token-type:device-secret"
+    "token_type": "urn:x-oath:params:oauth:token-type:device-secret",
+    "exp": 1628886628
```

Comment:
Intentionally expiring on Friday 13th? :)

Reply:
Lol didn't even realize that. what time we usually put, like 1 year from now?

Reply:
I think it's arbitrary, it was just kinda funny that it was Friday 13th..